### PR TITLE
Trlst 34 prevent document publish

### PR DESCRIPTION
**Pull request by markhigham**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -39,3 +32,9 @@ docker-cli-connect:
 
 build-docker-cli:
 		docker-compose -f docker-compose.yml build cli
+
+black:
```
Comment (rossmiller) [inline Makefile:36]: Ah! You have a compose file (my comment in https://github.com/uktrade/trade-remedies-caseworker/pull/3 is redundant. Could you refactor these to use the compose file to simplify them?

File: trade_remedies_public/core/base.py [hunk 2]
```diff
@@ -266,7 +272,11 @@ def get_submission_documents(self, documents=None, request_for_sub_org=False):
             "deficiency": parent_deficiency_documents,
             "issued": [doc for doc in documents if doc.get("issued")],
             "confidential": document_conf_index.get("true", []),
-            "non_confidential": document_conf_index.get("false", []),
+            "non_confidential": [
+                doc
```
Comment (rossmiller) [inline trade_remedies_public/core/base.py:276]: The formatting here is difficult to human parse.

File: trade_remedies_public/core/templatetags/download_link.py [hunk 3]
```diff
@@ -31,21 +22,38 @@ def download_link(context, document, all_downloadable=None):
...
-    #  Show docs as links if they are creted by this user or issued or not confidential
+
+    #  Show docs as links if they are created by this user or issued or not confidential
     if safe and downloadable:
         case = context.get("case") or {}
         reference = case.get("reference")
         submission_id = context.get("submission_id")
-        output = f"""<a href="/public/case/{reference}/submission/{submission_id}/document/{document.get('id')}/" class="link" target="_blank">{output}</a>"""
+        output = (
+            f"<a href=\"/public/case/{reference}/submission/{submission_id}/document/{document.get('id')}/\""
```
Comment (rossmiller) [inline trade_remedies_public/core/templatetags/download_link.py:54]: Single quotes so you can lose the escaping?

**Review by rossmiller — APPROVED** [review]
Looks good! Very minor amends if you fancy them.

File: trade_remedies_public/core/base.py (hunk 2 above)
Comment (markhigham) [reply trade_remedies_public/core/base.py:276]: I *think* this may have been black autoformatting

**Pull request merged**